testutil/logging.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Prometheus Authors
```

Comment:
Was expecting something like this to happen :)
```suggestion
// Copyright 2019 The Prometheus Authors
```